rct229/rules/section22/section22rule4.py

```diff
@@ -0,0 +1,148 @@
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.schema.config import ureg
+from rct229.utils.jsonpath_utils import find_all
+from rct229.utils.pint_utils import CalcQ
+from rct229.utils.std_comparisons import std_equal
+
+APPLICABLE_SYS_TYPES = [
+    "SYS-7",
+    "SYS-8",
+    "SYS-12",
+    "SYS-13",
+    "SYS-7B",
+    "SYS-8B",
+    "SYS-12B",
+    "SYS-13B",
+]
+NOT_APPLICABLE_SYS_TYPES = [
+    "SYS-11.1",
+    "SYS-11.2",
+    "SYS-11.1A",
+    "SYS-11.2A",
+    "SYS-11B",
+    "SYS-11C",
+]
+REQUIRED_OUTDOOR_HIGH_LOOP_SUPPLY_TEMP_RESET = 80 * ureg("degF")
```

Comment:
Now I think I understand why there was an issue with the measured temperatures like this.  I think the preferred way to express a measured temperature would be `ureg("80 degF")`. This avoids the multiplication that was causing the problem.

Reply:
Was this intention of the change to get rid of ```.to(ureg.kelvin)``` part in the ```rule_check``` method? I tested whether I can remove the Kelvin conversion part, However, I still got an error after the adjustment.

If the intention was to remove the Kelvin conversion part, I will dig more into this. If not, I will leave the adjustment as is. I'd appreciate it if @charliepnnl let me know about this!